extensions/amp-accordion/1.0/README.md

```diff
@@ -25,7 +25,17 @@ import {defineElement as defineBentoAccordion} from '@bentoproject/accordion';
 defineBentoAccordion();
 ```
 
-### Example: Include via `<script>`
+### Include via `<script>`
+
+```html
+<script type="module" src="https://cdn.ampproject.org/bento.mjs"></script>
+<script type="nomodule" src="https://cdn.ampproject.org/bento.js"></script>
+<script type="module" src="https://cdn.ampproject.org/v0/bento-accordion-1.0.mjs"></script>
+<script type="nomodule"  src="https://cdn.ampproject.org/v0/bento-accordion-1.0.js"></script>
```

Comment:
There's an extra space here (and in the other changed files):

```suggestion
<script type="nomodule" src="https://cdn.ampproject.org/v0/bento-accordion-1.0.js"></script>
```

Reply:
Also, `nomodule` should be an attribute not a type, right?

```suggestion
<script type="text/javascript" nomodule src="https://cdn.ampproject.org/v0/bento-accordion-1.0.js"></script>
```

Reply:
Fixed the space. [nomodule is an attribute](https://html.spec.whatwg.org/multipage/scripting.html#script-nomodule-example).